src/entries/popup/components/CommandK/useCommands.tsx

```diff
@@ -116,6 +116,16 @@ export const getStaticCommandInfo = (): CommandInfo => {
       toPage: PAGES.MY_WALLETS,
       type: SearchItemType.Shortcut,
     },
+    myContacts: {
+      actionLabel: actionLabels.view,
+      name: getCommandName('my_contacts'),
+      page: PAGES.HOME,
+      searchTags: getSearchTags('my_contacts'),
+      symbol: 'person.crop.circle.fill',
+      symbolSize: 16,
+      toPage: PAGES.MY_CONTACTS,
+      type: SearchItemType.Shortcut,
+    },
```

Comment:
Realised this change was removed in this PR https://github.com/rainbow-me/browser-extension/pull/1537. Added it back again here.